b2b-sample/guardio-insurance-sample-application/util/apiCall/settings/identityProvider/callListAllIdentityProviders.js

```diff
@@ -16,25 +16,22 @@
  * under the License.
  */
 
-import Cookie from 'js-cookie';
 import config from '../../../../config.json';
 import { getInternalApiRequestOptions } from '../../../util/apiUtil/getInteralApiRequestOptions';
 
-const subOrgId = Cookie.get("orgId");
-
 export default async function callListAllIdentityProviders(session) {
 
     try {
         const res = await fetch(
             `${config.WSO2IS_CLIENT_URL}/api/settings/identityProvider/listAllIdentityProviders`,
-            getInternalApiRequestOptions(session, subOrgId)
+            getInternalApiRequestOptions(session)
         );
 
         const data = await res.json();
 
         return data;
     } catch (err) {
-        
+
```

Comment:
Do we need these new lines?

Reply:
This have been fixed when the functionality of this moved to a new file

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/identityProvider/controllerListAllIdentityProviders/controllerCallListAllIdentityProviders.ts